latch/ldata/transfer/upload.py

```diff
@@ -250,25 +244,24 @@ def upload(
 
                         wait(chunk_futs)
 
-                        end_upload(
+                        _end_upload(
                             normalized,
                             res.upload_id,
                             [fut.result() for fut in chunk_futs],
                         )
 
     end = time.monotonic()
     total_time = end - start
-    if progress != Progress.none:
-        click.echo(
-            f"""{click.style("Upload Complete", fg="green")}
-
-{click.style("Time Elapsed: ", fg="blue")}{human_readable_time(total_time)}
-{click.style("Files Uploaded: ", fg="blue")}{num_files} ({with_si_suffix(total_bytes)})"""
-        )
+    if verbose:
```

Comment:
this isn't what `verbose` is for

Reply:
would add another param called `show_summary_message` or something, have the CLI always pass that in as true, and have `LPath` pass it in as false